src/Init/Data/BitVec/Basic.lean

```diff
@@ -354,7 +368,10 @@ end relations
 
 section cast
 
-/-- `cast eq x` embeds `x` into an equal `BitVec` type. -/
+/--
+If two natural numbers `n` and `m` are equal, then a bitvector of width `n` is also a bitvector of
+width `m`.
+-/
```

Comment:
```suggestion
/--
If two natural numbers `n` and `m` are equal, then a bitvector of width `n` is also a bitvector of
width `m`.

This is the preferred way to spell `(h : m = n) ▸ (x : BitVec m)`, due to available `simp` lemmas on `BitVec.cast` that simplify away more consistently than Lean's generic casting machinery.
-/
```

Shall we try and use the docstring to explain to people that this works better than the generic `h ▸  x`?

Reply:
Yes, I think so! I took your text and shortened it. Thank you!